toolkits/global/packages/global-card/HISTORY.md

```diff
@@ -1,5 +1,16 @@
 # History
 
+## 7.1.0 (2021-12-29)
```

Comment:
I think all of these changes should be lumped in to v7.0.0 because otherwise there is no way a user could install v7.0.0 if they tried to because it was never published.

Reply:
@benjclark Good point!